test/Orthogonal.hs

```diff
@@ -40,17 +40,17 @@ testSuite = testGroup "Orthogonal"
     ]
   ]
 
-prop_jacobi_de :: Rational -> Rational -> Property
-prop_jacobi_de a b = foldl' (.&&.) (property True) $
+prop_jacobi_de :: NonNegative Rational -> NonNegative Rational -> Property
+prop_jacobi_de (NonNegative a) (NonNegative b) = foldl' (.&&.) (property True) $
```

Comment:
Jacobi polynomials are defined for negative arguments, but our implementation is faulty. When a+b is integer, one can probably directly compute via the second formula at https://en.wikipedia.org/wiki/Jacobi_polynomials#Via_the_hypergeometric_function - two gamma functions degenerate to factorials and the ratio of other two is a Pochhammer symbol.